http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/BaseCloudServiceRuntimeMonitor.scala

```diff
@@ -111,6 +111,7 @@ abstract class BaseCloudServiceRuntimeMonitor[F[_]] {
             _ <- persistentDiskQuery
               .updateStatusAndIsFormatted(diskId, DiskStatus.Ready, FormattedBy.GCE, now)
               .transaction
+            _ <- persistentDiskQuery.updateNeedsResize(diskId, false).transaction
```

Comment:
why is this needed?

Reply:
once a runtime is ready (either the init or start-up script is complete) the disk will have been resized